javascript/statements.json

```
@@ -1377,6 +1377,65 @@
               "deprecated": false
             }
           }
+        },
+        "assertions": {
```

Let's give it a name with an underscore to keep with BCD's data structure:

```suggestion
        "import_assertions": {
```